lib/testing/src/lib/protractor/content-services/dialog/content-node-selector-dialog.page.ts

```diff
@@ -189,4 +190,10 @@ export class ContentNodeSelectorDialogPage {
     contentListPage(): DocumentListPage {
         return this.contentList;
     }
+
+    async checkUploadFromLocalTabIsDisabled() {
+        await BrowserVisibility.waitUntilElementIsPresent(this.uploadFromLocalTab);
+        const disabled = await BrowserActions.getAttribute(this.uploadFromLocalTab, 'aria-disabled');
+        expect(disabled).toBe('true');
```

Comment:
I am not sure if the page object is allowed to have expect function. Generally the real test should test the expected result.  Can we double check please?

Reply:
There are places that we have expectations inside the page like here https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/content-services/pages/search/date-range-filter.page.ts But I don't know to be honest, if it's a good practice or not

Reply:
@sivakumar414ram  is right, it is not a good practice to have expects in the page objects.

Reply:
Thank you both, I will update the expectation to be in the test 🙂